### PR TITLE
feat: Add headless template

### LINKED ISSUE #80: [feat] scaffolding of headless apps
It would be nice if there were an option to explicitly scaffold a headless app (i.e. without UI). Currently one is forced to select a UI framework and have UI components generated in any case and then must delete the UI folder manually. It's merely a question of convenience though.

### PR DESCRIPTION
**Pull request by c12i**
Closes #80

### CODE REVIEW

File: templates/svelte/web-app/package.json.hbs [hunk 1]
```diff
@@ -7,18 +7,18 @@
   ],
   "scripts": {
     "start": "AGENTS=${AGENTS:-2} BOOTSTRAP_PORT=$(port) SIGNAL_PORT=$(port) npm run network",
-    "network": "hc s clean && npm run build:happ && UI_PORT=$(port) concurrently \"npm start -w ui\" \"npm run launch:happ\" \"holochain-playground\"",
+    "network": "hc sandbox clean && npm run build:happ && UI_PORT=$(port) concurrently \"npm start -w ui\" \"npm run launch:happ\" \"holochain-playground\"",
```
Comment (matthme) [inline templates/svelte/web-app/package.json.hbs:10]: I like this change, makes it more explanatory.

File: src/scaffold/web_app/uis.rs [hunk 2]
```diff
@@ -90,10 +109,11 @@ impl TryFrom<&FileTree> for UiFramework {
 impl std::fmt::Display for UiFramework {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let str = match self {
-            UiFramework::Vanilla => "vanilla",
-            UiFramework::Lit => "lit",
-            UiFramework::Svelte => "svelte",
-            UiFramework::Vue => "vue",
+            UiFramework::Vanilla => "vanilla".yellow(),
+            UiFramework::Lit => "lit".blue(),
```
Comment (matthme) [inline src/scaffold/web_app/uis.rs:113]: The blue is hardly visible in my Ubuntu terminal:
image

I wonder whether `bright_blue` might be better.

**Review by matthme — CHANGES_REQUESTED** [review]
Just some minor change proposals but looks good in general - I'm going to use that in the future, thanks!

File: templates/headless/web-app.instructions.hbs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+This is a headless skeleton for a hApp. 
+
```
[the remark this answers is not in the document]
Comment (c12i) [reply templates/headless/web-app.instructions.hbs:2]: Good idea

File: src/scaffold/web_app/uis.rs (hunk 2 above)
Comment (c12i) [reply src/scaffold/web_app/uis.rs]: Fixed

**Review by matthme — APPROVED** [review]

**Pull request merged**